<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sado_qtdialog</class>
 <widget class="QDialog" name="sado_qtdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kewe_qtdialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_left" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="parameters">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <row>
         <property name="text">
          <string>b</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>c</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>end_time</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>erasure_method</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>eta</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>histbinp</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>histbinq</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>histbinx</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>next_generation_method</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>output_filename</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>output_freq</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>p0</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>pop_size</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>q0</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>sc</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>se</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>seed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>sk</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>sm</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>sq</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>sv</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>use_initialization_bug</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>x0</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_branching">
        <property name="text">
         <string>Set &amp;branching parameters</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>&amp;Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_branching">
        <property name="text">
         <string>Show branching!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="right_scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="results">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>339</width>
        <height>561</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>male sexual trait</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ribi::QtSurfacePlotWidget" name="male_sexual_trait" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Female sexual preference</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ribi::QtSurfacePlotWidget" name="female_preference" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ecological trait</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ribi::QtSurfacePlotWidget" name="eco_trait" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>New stuff [TODO]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ribi::QtSurfacePlotWidget</class>
   <extends>QWidget</extends>
   <header>../SurfacePlotter/qtsurfaceplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
